js/views/settings.js

```diff
@@ -176,7 +180,21 @@ app.views.Settings = (function() {
 				errors.push(app.i18n.t('settings.at-least-one-crypto-currency-required'));
 			}
 
-			return errors;
+
+			_.each(data.acceptCryptoCurrencies, _.bind(function(currency) {
+
+				app.paymentMethods[currency].validateAsync(data, function(error, validatedData) {
```

Comment:
Validation should only return errors (if there are any) - not change the data it is checking.